subsys/bluetooth/host/att.c

```diff
@@ -3832,6 +3832,27 @@ uint16_t bt_att_get_mtu(struct bt_conn *conn)
 	return mtu;
 }
 
+uint16_t bt_att_get_uatt_mtu(struct bt_conn *conn)
+{
+	struct bt_att_chan *chan, *tmp;
+	struct bt_att *att;
+
+	att = att_get(conn);
+	if (!att) {
+		return 0;
+	}
+
+	SYS_SLIST_FOR_EACH_CONTAINER_SAFE(&att->chans, chan, tmp, node) {
+		if (!bt_att_is_enhanced(chan)) {
+			return bt_att_mtu(chan);
+		}
+	}
+
+	/** There must be an UATT channel per connection */
+	k_oops();
+	CODE_UNREACHABLE;
```

Comment:
We should avoid `CODE_UNREACHABLE`. It's very unsafe (a better name would be `INVOKE_UNDEFINED_BEHAVIOR`), so we need a good reason for using it.

```suggestion
	return 0;
```